auth.go

```diff
@@ -372,12 +376,20 @@ func setGroup(icentral *imqsauth.ImqsCentral, groupName string, roles []string)
 	return imqsauth.ModifyGroup(icentral, imqsauth.GroupModifySet, groupName, perms)
 }
 
-func createUser(icentral *imqsauth.ImqsCentral, options map[string]string, email, password, firstname, lastname string) bool {
-	if _, e := icentral.Central.CreateUserStoreIdentity(email, "", firstname, lastname, options["mobile"], password); e == nil {
-		fmt.Printf("Created user %v\n", email)
-		return true
+func createUser(icentral *imqsauth.ImqsCentral, options map[string]string, email string, password string) bool {
+
+	// We validate the email address from the command line here. Email addresses are mandatory identifiers for IMQS users.
+	matched, _ := regexp.MatchString("^([\\w-]+(?:\\.[\\w-]+)*)@((?:[\\w-]+\\.)*\\w[\\w-]{0,66})\\.([a-z]{2,6}(?:\\.[a-z]{2})?)$", email);
```

Comment:
If there's a stray semicolon, it means you're not using gofmt. Use sublimetext + gosublime